docker-compose-prod.yml

```diff
@@ -15,6 +15,8 @@ services:
       file: ./common-services.yml
       service: shogun-geoserver
     image: docker.osgeo.org/geoserver:2.25.3
+    environment:
+      PROXY_BASE_URL: ${GEOSERVER_PROXY_BASE_URL}
```

Comment:
Where is the variable `GEOSERVER_PROXY_BASE_URL` coming from? Do we need to add it to the `setEnvironment.sh` skript?

Have you also checked, that [this](https://docs.geoserver.org/stable/en/user/configuration/globalsettings.html#use-headers-for-proxy-url) option is checked in your environment? I'm just curious since this env should then not be necessary.